src/pages/Home/PillChallenge/PillManagePage.tsx

```diff
@@ -0,0 +1,371 @@
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                handleEatingDay(e, "wednesday")
+              }
+              defaultChecked={newPill.weeklyIntakeFrequency.wednesday}
+            />
+            <label htmlFor="wed" className={s.bigButton}>
+              수
+            </label>
+            <input
+              type="checkbox"
+              id="thu"
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                handleEatingDay(e, "thursday")
+              }
+              defaultChecked={newPill.weeklyIntakeFrequency.thursday}
+            />
+            <label htmlFor="thu" className={s.bigButton}>
+              목
+            </label>
+            <input
+              type="checkbox"
+              id="fri"
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                handleEatingDay(e, "friday")
+              }
+              defaultChecked={newPill.weeklyIntakeFrequency.friday}
+            />
+            <label htmlFor="fri" className={s.bigButton}>
+              금
+            </label>
+            <input
+              type="checkbox"
+              id="sat"
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                handleEatingDay(e, "saturday")
+              }
+              defaultChecked={newPill.weeklyIntakeFrequency.saturday}
+            />
+            <label htmlFor="sat" className={s.bigButton}>
+              토
+            </label>
+            <input
+              type="checkbox"
+              id="sun"
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                handleEatingDay(e, "sunday")
+              }
+              defaultChecked={newPill.weeklyIntakeFrequency.sunday}
+            />
+            <label htmlFor="sun" className={s.bigButton}>
+              일
+            </label>
+
+          </div>
+          
+        </div>
+
+          <div className={s.detailDiv}>
+            <div className={s.messengerAlarmHeader}>
+              <span className={s.detailTitle}>키키오톡 알림 시간</span>
+              <button
+                type="button"
+                onClick={() => setModal(true)}
+                className={s.plusButton}
+              >
+                <img src={plusIconImg} alt="" />
+              </button>
+            </div>
+            <div className={s.messengerAlarmBody}>
+              <span></span>
+            </div>
+          </div>
+          <button
+            type="button"
+            className={s.completeButton}
+            onClick={() => handleChanges()}
+          >
+            완료
+          </button>
+
+          <div className={s.bottomBarCover}></div>
+
+          <div className={s.messengerAlarmBody}>
+            <span></span>
+          </div>
+        </div>
+
+        <div className={s.bottomBarCover}></div>
+      </div>
+      `
+      {modal === true ? (
+        <AlarmTimeInputModal modal={modal} setModal={setModal} />
+      ) : null}
```

Comment:
&& 연산자를 사용하는 것이 가독성이 더 좋을 것 같습니다.
```suggestion
      {modal && <AlarmTimeInputModal modal={modal} setModal={setModal} />}
```